about/index.markdown

```diff
@@ -5,92 +5,59 @@ date: 2013-11-20 04:37:44+00:00
 
+## Citing MoveIt!
 
-### Citing MoveIt!
-
-
-We are working on a technical publication that will be a better resource to cite MoveIt! in the near future. In the meantime, please use the following citation for MoveIt!
-
-
+Please use the following citation for MoveIt!
 
 
   * Ioan A. Sucan and Sachin Chitta, "MoveIt!", [Online] Available: [http://moveit.ros.org](http://moveit.ros.org)
 
-
-### Website 
-The website is maintained by Sachin Chitta and Praveen Singh.
-
-### License
+## License
 
 
 MoveIt! is open source and released under the BSD License. Each individual file in the MoveIt! source code contains a copy of the license.
 
+## Maintainers
 
-### Source Code
-
-
-MoveIt! code is hosted on github in the [ros-planning organization](http://github.com/ros-planning) in the following repos:
-
-
-
-
-  * [moveit_core](https://github.com/ros-planning/moveit_core) - Core C++ libraries
-
-
-  * [moveit_ros](https://github.com/ros-planning/moveit_ros) - ROS components
-
-
-  * [moveit_msgs](https://github.com/ros-planning/moveit_msgs) - MoveIt! messages
-
-
-  * [moveit_setup_assistant](https://github.com/ros-planning/moveit_setup_assistant) - MoveIt! Setup Assistant
-
+The MoveIt! project is currently maintained by the following contributors with commit-access:
 
-  * [moveit_planners](https://github.com/ros-planning/moveit_planners) - MoveIt! Planning libraries
+Name | Organization | GitHub ID | Project Specialities
+------------ |:------------- |-------------|-------------|
+Michael Ferguson | Fetch Robotics | [mikeferguson](https://github.com/mikeferguson) | Releasing and Regressions
+Dave Coleman | Correll Lab, CU Boulder | [davetcoleman](https://github.com/davetcoleman) | Reviewing PRs
+Robert Haschke | CITEC, Bielefeld University | [rhaschke](https://github.com/rhaschke) | Fixing Bugs, Reviewing PRs
+Michael Görner | University of Hamburg | [v4hn](https://github.com/v4hn) | Fixing Bugs, New Features
+Isaac IY Saito | Tokyo Open Source Robotics | [130s](https://github.com/130s) | Releasing and Continuous Integration
+Sachin Chitta | Kinema Systems | [sachinchitta](https://github.com/sachinchitta) | Original Founder & Author of MoveIt!
+Ioan Sucan | Google X | [isucan](https://github.com/isucan) | Original Founder & Author of MoveIt!
+Ian McMahon | Rethink Robotics | [IanTheEngineer](https://github.com/IanTheEngineer) | Releasing
+Gijs van der Hoorn | Delft Univ. of Tech / ROS-I | [gavanderhoorn](https://github.com/gavanderhoorn) | IKFast Plugin
+Jorge Nicho | SwRI / ROS-I | [jrgnicho](https://github.com/jrgnicho) | IKFast Plugin
+Christian Dornhege | University of Freiburg | [dornhege](https://github.com/dornhege) | PR2 Code Base
 
+## Maintainer Alumni
 
-  * [moveit_commander](https://github.com/ros-planning/moveit_commander) - MoveIt! Commander
+We would like to acknowledge past maintainers of MoveIt!
 
+Name             | GitHub ID |
+------------     |:------------- |
+Dave Hershburger | hersh |
+Acorn Pooley | N/A |
```

Comment:
So they do not have access to the MoveIt repositories anymore?
At least @hersh  is still a member of ros-planning afaics